pootle/core/search/backends/elasticsearch.py

```diff
@@ -75,6 +92,10 @@ def search(self, unit):
             }
         )
 
+        # Sometimes urllib seems to return empty string so handle this here
```

Comment:
im minded to keep it as is in terms of comment - this is where i want this information, rather than having to check through the commit history